apps/crn-server/test/data-providers/research-outputs.data-provider.test.ts

```diff
@@ -666,13 +667,11 @@ describe('ResearchOutputs data provider', () => {
         withTeams: true,
       };
 
-      beforeEach(() => {
+      test('Should pass the pagination parameters as expected', async () => {
```

Comment:
maybe rather than remove the beforeEach and repeat the mocks all the time just have another nested describe for all tests that do not pass the search parameter and then another one for the ones that do?